ui/src/submissions/authors/containers/AuthorSubmissionPage.jsx

```diff
@@ -41,6 +45,7 @@ class AuthorSubmissionPage extends Component {
                 error={error}
                 onSubmit={this.onSubmit}
                 initialFormData={initialFormData}
+                overrideSchema={extraSchemaForNewAuthor}
```

Comment:
Am I missing something all this change will always (create & update) will override the schema?

Reply:
it does not, actually I think `overrideSchema` is not a good name, because what does is `yup.concat`. We could call it `extendSchema` or `extraSchema` sth like that.

Reply:
``extendSchema`` makes more sense 👍